lib/messages/bind_request.js

```diff
@@ -71,13 +71,18 @@ BindRequest.prototype._toBer = function (ber) {
   return ber;
 };
 
-BindRequest.prototype._json = function (j) {
+BindRequest.prototype._json = function (j, safePrint) {
```

Comment:
Can you show me some instances where this method is invoked? Are invokers already passing this second parameter and it just wasn't implemented? Or is this a completely new parameter that would need to be passed from somewhere?